xds/src/test/java/io/grpc/xds/EdsLoadBalancer2Test.java

```diff
@@ -493,16 +489,30 @@ private Long populateMaxConcurrentRequests(PriorityLbConfig config, String prior
     return clusterImplConfig.maxConcurrentRequests;
   }
 
-  private PolicySelection populateLeafLbPolicy(PriorityLbConfig config, String priority,
-      Locality locality) {
+  @Test
+  public void configUpdate_changeTlsContext_propagateToChildLb() {
+    deliverSimpleClusterLoadAssignment(EDS_SERVICE_NAME);
+    FakeLoadBalancer childBalancer = Iterables.getOnlyElement(childBalancers);
+    PriorityLbConfig childLbConfig = (PriorityLbConfig) childBalancer.config;
+    assertThat(populateTlsContext(childLbConfig, "priority1")).isNull();
+
+    UpstreamTlsContext upstreamTlsContext =
+        CommonTlsContextTestsUtil.buildUpstreamTlsContextFromFilenames(
+            CommonTlsContextTestsUtil.CLIENT_KEY_FILE,
+            CommonTlsContextTestsUtil.CLIENT_PEM_FILE,
+            CommonTlsContextTestsUtil.CA_PEM_FILE);
+    EdsConfig config = new EdsConfig(CLUSTER, EDS_SERVICE_NAME, LRS_SERVER_NAME, 100L,
+        upstreamTlsContext, weightedTarget, roundRobin);
+    deliverConfig(config);
+    assertThat(Iterables.getOnlyElement(childBalancers)).isSameInstanceAs(childBalancer);
+    childLbConfig = (PriorityLbConfig) childBalancer.config;
+    assertThat(populateTlsContext(childLbConfig, "priority1")).isEqualTo(upstreamTlsContext);
+  }
+
+  private UpstreamTlsContext populateTlsContext(PriorityLbConfig config, String priority) {
```

Comment:
Why is this called `populate...`? It is misleading. It is just extracting a tlsContext from the child policy of a priority. `extractTlsContext` would be more appropriate.

Reply:
Well this is private and in test, and the EDS LB policy will soon be replaced by the cluster_resolver LB policy (#7685 and #7722 migrates to it, then this will be deleted). So we probably don't bother picking these things. 😄